client/src/components/Projects/ProjectsPage.js

```diff
@@ -355,6 +359,27 @@ const ProjectsPage = ({ account, contentContainerRef }) => {
     )
       return false;
 
+    if (
```

Comment:
The way this function is written, each conditional should test whether the project should be excluded, and return false if it should be excluded.  So you can't return true until all of the filter conditions are met, or you would be ignoring all the subsequent conditional tests.  So, for example, lines 367-371 should be  of "inverted" to return false if there is an endDateCreated and the project's dateCreated is after the endDateCreated.  Likewise for the lines 377-381

Reply:
Thanks for the notes, @entrotech! I'll make the suggested changes and get this back up shortly.